internal/service/fsx/ontap_volume.go

```diff
@@ -93,6 +93,12 @@ func ResourceOntapVolume() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"snapshot_policy": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
Should this be computed for should the default value be "default"

I think we can add minimal validation according to docs:
```

	// Specifies the snapshot policy for the volume. There are three built-in snapshot
	// policies:
	//
	//    * default: This is the default policy. A maximum of six hourly snapshots
	//    taken five minutes past the hour. A maximum of two daily snapshots taken
	//    Monday through Saturday at 10 minutes after midnight. A maximum of two
	//    weekly snapshots taken every Sunday at 15 minutes after midnight.
	//
	//    * default-1weekly: This policy is the same as the default policy except
	//    that it only retains one snapshot from the weekly schedule.
	//
	//    * none: This policy does not take any snapshots. This policy can be assigned
	//    to volumes to prevent automatic snapshots from being taken.
	//
	// You can also provide the name of a custom policy that you created with the
	// ONTAP CLI or REST API.
```

Reply:
Validation is complicated by this part. Users can put an arbitrary policy name here. 

```
// You can also provide the name of a custom policy that you created with the
// ONTAP CLI or REST API.
```

Reply:
Fair enough regrading validation. i dont think this should be computed though as drift detection wont work

Reply:
I've run through the full lifecycle (apply/update/destroy) with several volumes using many combinations of resource configuration and I haven't noticed any issues with drift detection so far. Is there a specific case or sequence of events where drift detection won't work that I can test? Thanks!

Reply:
Removed computed and added Default